intents/swap/src/lost_found.rs

```diff
@@ -17,15 +14,18 @@ impl LostFound for SwapIntentContractImpl {
 
```

Comment:
In the method `lost_found` replace:
`assert_eq!(env::attached_deposit(), NearToken::from_yoctonear(1));`
to 
`near_sdk::assert_one_yocto();`
And as I know they don't recommend using `assert`s in the contract code. There is analog: `near_sdk::require!`.